src/Umbraco.Cms.Api.Management/Controllers/Document/References/AreReferencedDocumentController.cs

```diff
@@ -28,16 +28,16 @@ public ItemsTrackedReferenceController(ITrackedReferencesService trackedReferenc
     ///     Used when bulk deleting content/media and bulk unpublishing content (delete and unpublish on List view).
     ///     This is basically finding children of relations.
     /// </remarks>
-    [HttpGet("item")]
+    [HttpGet("are-referenced")]
```

Comment:
I think `referenced-by` would be a more meaningful route?

Reply:
This endpoint returns which of the items you pass in are used by other items in a dependent way. So whether they are a reference on, aka are referenced by, another item